app/web/features/auth/FeaturePreview.tsx

```diff
@@ -45,6 +46,7 @@ export default function FeaturePreview() {
         <Alert severity="error">{accountInfoError.message}</Alert>
       ) : (
         <>
+          <PushNotificationPermission className={classes.section} />
```

Comment:
Should this be called `PushNotificationSettings` or something? It's not just the permission anymore.

Reply:
Changed the name now!